scripts/fixup.gd

```diff
@@ -0,0 +1,21 @@
+tool
+extends EditorScript
+
+func _run():
+	var dir : Directory = Directory.new()
+	dir.open("res://addons/material_maker/nodes")
+	dir.list_dir_begin()
+	var file_name = dir.get_next()
+	var loader = load("res://addons/material_maker/engine/loader.gd").new()
+	while file_name != "":
+		if file_name.ends_with('.mmg'):
+			print("converting "+file_name)
+			var file_path : String = "res://addons/material_maker/nodes".plus_file(file_name)
+			var file : File = File.new()
+			if file.open(file_path, File.READ) == OK:
+				var generator = loader.string_to_dict_tree(file.get_as_text())
+				file.close()
+				file.open(file_path, File.WRITE)
+				file.store_string(loader.dict_tree_to_string(generator)+"\n")
```

Comment:
It's better to call `loader.save_gen()` here since that has code to clean up the generator structure, and maybe also `loader.load_gen()` above so it's balanced 🙂

Reply:
This is not as easy as it seems since actually loading a node relies on globals that are not available in editor mode.

Reply:
Oh, never mind, it works.